sycl/doc/DeviceGlobal.md

```diff
@@ -359,6 +359,11 @@ design document][5] for more details on this topic.
 
 [5]: <SpecializationConstants.md>
 
+### Changes to the DPC++ driver
+
+A new command line argument, `--device-globals` must be passed to the 
+`sycl-post-link` tool to enable processing device global variables.
+
```

Comment:
Why do we want this to be optional?  Does the `dpcpp` compiler driver always pass `--device-globals` to `sycl-post-link`?  What about the `clang` driver (used for the open source project)?

Reply:
Currently neither `dpc++` nor `clang` use this command line argument. I've added it to the `sycl-post-link` in order to be aligned with the other arguments, `sycl-post-link` enables only those parts of its functionality the command line arguments for what present, specialization constants, esimd lowering, etc. This is suitable for testing, for example.

As I know, the drivers currently pass all the defined command line arguments when `-fsycl` are in the command line, so they are required also to pass `--device-globals` too.

Reply:
Thanks for explaining.  I'm not sure it is necessary add this information to the design document, but I am not opposed.